README.md

```diff
@@ -14,8 +14,59 @@ the packages individually, for example:
     make -C debs/xenial/archivematica
     make -C debs/xenial/archivematica-storage-service
 
-### Test packages
+#### Using parameters
 
+Most makefiles support parameters in the format of environment variables. They 
+may be a bit different between packages, but the most common are:
+    BRANCH
+    VERSION
+    RELEASE
+
+So in order to build an specific branch / version, this command can be used from
+the folder of the package we want to build:
+
+    make BRANCH=qa/1.x VERSION=1.7.0 RELEASE=rc5
+
+Have in mind that the Makefiles are a bit recursive, they will invoke docker, 
+mount the current folder, and run the deb-build target.
```

Comment:
I think "Keep in mind" sounds more natural than "Have in mind". Also, I don't really understand what you mean by "recursive" here. What is it you're trying to convey to the user here. (Maybe I just don't know enough about this domain.)

Reply:
It might not be the better explanation. The Makefile has a job for running docker, and other job to build the package inside the docker container that was previously created